frontend/src/FullPageRoutes/FAQ.tsx

```diff
@@ -0,0 +1,169 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  makeStyles,
+  Typography,
+  Grid,
+} from "@material-ui/core"
+import * as React from "react"
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore"
+
+const useStyles = makeStyles({
+  page: {
+    margin: "1px",
+    padding: "50px",
+    backgroundColor: "rgb(238,235,228)",
+  },
+  topBar: {
+    margin: "1px",
+    padding: "50px",
+    backgroundColor: "rgb(238,235,228)",
+  },
+  bottomBar: {
+    margin: "2px",
+    padding: "110px",
+  },
+})
+
+export default function FAQ(): JSX.Element {
+  const classes = useStyles()
+  return (
+    <>
+      <Grid
+        container
+        direction="column"
+        alignItems="center"
+        className={classes.topBar}
+      >
+        <Grid item justify="center">
+          <Typography variant="h4">Frequently Asked Questions</Typography>
+          <Typography variant="h5">(FAQs)</Typography>
+        </Grid>
+
+        <Grid item>
+          <Grid
+            container
+            justify="center"
+            alignItems="center"
+            spacing={1}
+            className={classes.bottomBar}
+          >
+            <Grid item>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>How can I sign up for Conteo?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    To sign up, go to our homepage and select the sign up
+                    options. You would then be asked to provide your email and
+                    password.
+                  </Typography>
+                </AccordionDetails>
+              </Accordion>
+              <Accordion>
+                <AccordionSummary
+                  expandIcon={<ExpandMoreIcon />}
+                  aria-controls="panel1a-content"
+                >
+                  <Typography>Who are penpals?</Typography>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <Typography>
+                    Penpals are anyone you are matched to.
```

Comment:
"Penpals are other users that are in your penpal list. You can send video letters to them and they can send video letters to you. You can only be penpals with people that you approved in your match list, and who approved you. You can remove a penpal from your list if you don't want to correspond with them anymore."